examples/browser/package.json

```diff
@@ -25,7 +25,6 @@
     "@theia/file-search": "1.34.0",
     "@theia/filesystem": "1.34.0",
     "@theia/getting-started": "1.34.0",
-    "@theia/git": "1.34.0",
```

Comment:
We should revert - we will describe how to test in the pull-request description instead.

Reply:
Done. Yeah, makes more sense that way.